blog/2022-11-21-golang-memory-dump.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Profiling golang in kubernetes
+description: Profiling golang in kubernetes using Kubernetes debug.
+authors: nissessenap
+tags:
+  - pprof
+  - kubernetes
+  - debug
+keywords:
+  - pprof
+  - kubernetes
+  - debug
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+In kubernetes 1.23 the new API ephemeral containers went in to beta and in 1.25 it became stable.
+
+Ephemeral containers or debug containers which is also known as, makes it's possible to attach a container to a already running pod without restarting it.
+
+This is extremely useful when you want to debug your application since they can contain debugging tools that you don't to have in your application container.
+
+In this post I thought we could go trough how to profile a running container in Kubernetes.
```

Comment:
I think I would have joined all above four paras into one. It will make for a better blurb.